GWXtreme/eos_prior.py

```diff
@@ -0,0 +1,338 @@
+# Copyright (C) 2017 Daniel Wysocki
+# Assimilated from Wysocki D., O'Shaughnessy R., Bayesian Parametric Population Models, 
+# 2017–, bayesian-parametric-population-models.readthedocs.io
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 2 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+
+import typing
+import numpy
+import scipy.stats
+import h5py
+
+
+largest_ns_mass = 1.97
+
+def eos_p_of_rho(rho,eos):
```

Comment:
For uniformity, please add docstring describing the function like is done in the other codes in the package.